quickstarts/angular1/01-login.md

```diff
@@ -0,0 +1,222 @@
+
+![App Dashboard](/media/articles/angularjs/app_dashboard.png)
+
+TODO: Find out how to configure angular-sample URL
+<%= include('../_includes/_package', {
+  pkgRepo: 'auth0-angular',
+  pkgBranch: 'master',
+  pkgPath: 'examples/widget-redirect',
+  pkgFilePath: null,
+  pkgType: 'js'
+}) %>_
+
+::: panel-info Running the Sample
+At any point in time, you can run the sample with `http-server`. First install with `npm install -g http-server` then run `http-server` on the root directory of the sample.
+:::
+
+::: panel-info Default Configuration
+Every instance of the seed project comes configured with your `Default App` credentials. Awesome right?
+:::
+
+### Configure Callback URLs
+
+Callback URLs are URLs that Auth0 invokes after the authentication process. It is especially useful in OAuth. Callback URLs can be manipulated on the fly and that could be harmful.
+
+For security reasons, you will need to add your application's URL in the app's `Allowed Callback URLs`. This will enable Auth0 to recognize the URLs as valid. If omitted, authentication will not be successful for the app instance.
+
+![Callback error](/media/articles/angularjs/callback_error2.png)
+
+### Authentication
+
+We will stick with a simple structure for this section:
+```bash
+|---app.js
+|---index.html
+```
+
+Going through these following sub-steps, you'll be able to replicate this process on your own Angular applications.
+
+#### 1: Setup Scripts, Viewport and Basic Style
+Some JavaScript dependencies are required for Auth0 to work as expected in an Angular app. Include the dependencies' scripts in yout `index.html`:
+
+${snippet(meta.snippets.dependencies)}
+
+These may seem like a lot of dependencies, but each one has a very important function. Go through and figure out if each package works for your use case. Feel free to strip off the unnecessary packages.
+
+ - **lock** is the default authentication widget provided by Auth0. It is completely optional but I suggest you stick to it as an Auth0 newbie.
+ - **angular** is Angular's main library which you are building the application on.
+ - **angular-cookies** is Angular's wrapper for managing client cookies.
+ - **angular-route** is used to mange SPA routes in Angular application.
+ - **auth0-angular**: Auth0's SDK for Angular. Exposes most of the useful methods for authentication
+ - **angular-storage**: A `localStorage` and `sessionStorage` wrapper create with love by Auth0 team.
+ - **angular-jwt**: Angular service that makes using JWT easy in Angular apps
+
+ ::: panel-info Note
+ Some of the includes like angular-route and angular-storage will not be used in this step but will be useful in subsequent steps
+ :::
```

Comment:
Should we just add these when they are needed? What if a user doesn't follow all steps? They will have extra dependencies.